### ISSUE TITLE
How do i enable repairing and disenchanting?

### ISSUE THREAD

**Issue by Razgriz-3** [Epoxide-Software/Enchanting-Plus#172]
I'm using the most current version of the mod available on curseforge (1.12.2-5.0.176) and I don't have the options to repair anything on the advanced table in game. I don't see any options for enabling disenchanting in the config file either. I'm also using the latest 1.12 version of bookshelf. Any idea what I'm doing wrong?

My config file:

# Configuration file

blacklist {
    # A blacklist of enchantments that are not available in E+. Format is just enchantmentid. [default: ]
    S:blacklistedEnchantments <
     >

    # A blacklist of items that can't be enchanted with this mod. Format is itemid#meta [default: ]
    S:blacklistedItems <
     >
}

general {
    # The base cost to use for the enchantment formula.  [range: 1 ~ 1024, default: 45]
    I:baseCost=45

    # A number used when calculated enchantment cost. This number is treated as a % based factor. 0.30 = 30% of the original cost. 1.5 = 150% of the original cost. [range: 0.0 ~ 1024.0, default: 1.0]
    S:costFactor=1.0

    # A factor used to make curse enchantments like vanishing cost more to apply. By default they cost 3X more. [range: 0.0 ~ 1024.0, default: 3.0]
    S:curseFactor=2.0

    # The amount of enchantment power a floating book should give. Bookshelfs have 1 power. [range: 0.0 ~ 1024.0, default: 1.0]
    S:floatingBookPower=1.0

    # A factor used to make treasure enchantments like mending cost more to apply. By default they cost 4X more. [range: 0.0 ~ 1024.0, default: 4.0]
    S:treasureFactor=2.0
}

**Comment by democat3457** [comment]
https://github.com/Epoxide-Software/Enchanting-Plus/issues/143#issuecomment-563511706

**Comment by Coinzs** [comment]
Yeah, I'm having the same problem as well. I've tried putting the code at the end and that did not work, then tried deleting the dots which did not work as well.
